qiita_db/support_files/patches/42.sql

```diff
@@ -0,0 +1,15 @@
+-- Nov 19, 2016
+-- Adding provenance parameter to validate commands
+
+DO $do$
+DECLARE
+    cmd  RECORD;
+BEGIN
+    FOR cmd IN
+        SELECT command_id FROM qiita.software_command WHERE name = 'Validate'
+    LOOP
+        INSERT INTO qiita.command_parameter (command_id, parameter_name, parameter_type, required, default_value)
```

Comment:
Note that this is adding a parameter called "provenance" to the validate command. The command itself doesn't use it but it is used internally in Qiita to differentiate a validate command that has been generated as a result of the user uploading data to Qiita from a validate command that has been generated as a result of an "artifact transformation" job finishing. This doesn't really contain the provenance of the artifact, so no need to add to the already created artifacts.

Note that the provenance of the artifacts is stored on the DAG structure in the DB.